sources/config_reader.c

```diff
@@ -294,6 +296,36 @@ od_config_reader_symbol(od_config_reader_t *reader, char symbol)
 	return false;
 }
 
+static bool
+od_config_reader_percentiles(od_config_reader_t *reader, char *value, double **percentiles, int *count)
```

Comment:
And all this code just for splitting a string  to a double array.

Reply:
Also, it's possible to use strsep http://man7.org/linux/man-pages/man3/strsep.3.html
But I guess it will not really simplify this function

Reply:
It will be better to return double* array(but as I see it's a common pattern here) 
or at least pass it as double* instead of double**

Reply:
It's common pattern in config parsing, I don't think to rewrite it fits into this PR.